pkg/crc/constants/constants.go

```diff
@@ -83,32 +83,30 @@ func GetAdminHelperURL() string {
 	return GetAdminHelperURLForOs(runtime.GOOS)
 }
 
-func defaultBundleForOs(preset crcpreset.Preset) map[string]string {
```

Comment:
I am going to use this helper function to get bundle name when provided by http/https or docker scheme. I am changing the signature of this `func defaultBundleForOs(preset crcpreset.Preset, version string) map[string]string` so that I can add another function in constant something like. As of now our images are like `quay.io/crcont/ocp-bundle:4.11.0` so only `tag` is useful to create the bundle name for respective preset.

```golang
func GetBundleNameFromBundlePath(preset crcpreset.Preset, bundlePath string) string {
	switch {
	case strings.HasPrefix(bundlePath, "http://"), strings.HasPrefix(bundlePath, "https://"):
		return filepath.Base(bundlePath)
	case strings.HasPrefix(bundlePath, "docker://"):
		tag := strings.Split(bundlePath, ":")[2]
		bundles := defaultBundleForOs(preset, tag)
		return bundles[runtime.GOOS]
	default:
		return filepath.Base(bundlePath)
	}
}
```